doc/webui/README.md

```diff
@@ -0,0 +1,66 @@
+# elektra web
+
+_a web user interface (Web UI) to remotely manage multiple elektra instances_
+
+
+## Overview
+
+Elektra web consists of multiple components:
+
+ * (multiple) servers running an elektra daemon (`elektrad`)
+ * a single cluster management server to communicate with the elektra daemons (`clusterd`)
```

Comment:
I agree that it would make sense to have a separate document that goes into details of the architecture. Here I'm just giving a small overview to make it easier to understand the API.